### PR TITLE
PY: Hero's Day and 2017 adjustments

### PR DESCRIPTION
**Pull request by gerryster**
This adds Hero's Day to Paraguay as well as updating a few holidays in 2017 which were moved per a presidential decree.

### CODE REVIEW

File: data/countries/PY.yaml [hunk 1]
```diff
@@ -12,6 +15,14 @@ holidays:
     days:
       01-01:
         _name: 01-01
+      03-01:
+        disable:
+          - '2017-03-01'
```
Comment (gerryster) [inline data/countries/PY.yaml]: After reading through the YAML specification, this appeared to be a the preferred method of changing the date on a one-off year. Please let me know if there is a better way.

Comment (commenthol) [reply data/countries/PY.yaml]: This is the only method. The reason is that some calendars offer more than one date per gregorian year.

**Comment by commenthol** [comment]
Thanks

File: data/countries/PY.yaml (hunk 1 above)
Comment (gerryster) [reply data/countries/PY.yaml]: Ok, that makes sense. However, suppose that the Paraguayan government adjusted this holiday in 2020 by a day or two. How would that be represented here?

Comment (commenthol) [reply data/countries/PY.yaml]: Please check https://github.com/commenthol/date-holidays/blob/5deac678e61afbe2b72d2b7005501c17291f6e15/data/countries/GY.yaml#L44

Comment (gerryster) [reply data/countries/PY.yaml]: Oh, nice! Tuples.

**Pull request merged**